charmcraft/commands/store/client.py

```diff
@@ -47,6 +47,34 @@ def build_user_agent():
     )
 
 
+class AnonymizedClient(craft_store.http_client.HTTPClient):
+    """Lightweight layer that access public store data."""
+
+    def __init__(self, api_base_url, storage_base_url, ephemeral=False):
+        self.api_base_url = api_base_url.rstrip("/")
+        self.storage_base_url = storage_base_url.rstrip("/")
+        del ephemeral  # Placeholder
```

Comment:
? what is this for? It looks to me like the `ephemeral` parameter is auth-related, so this class doesn't need the parameter at all right?

Reply:
also this init is missing type annotations